modules/gdscript/tests/scripts/runtime/features/multilevel_methods.gd

```diff
@@ -0,0 +1,120 @@
+class A:
+	func _notification(what: int) -> void:
+		if what == 10:
+			print("A")
+
+	func _get_property_list() -> Array[Dictionary]:
+		var ret: Array[Dictionary]
+		ret.append({ "name": "password", "type": TYPE_INT })
+		return ret
+
+	func _set(property: StringName, _value: Variant) -> bool:
+		if property == &"password":
+			# Should not be reached.
+			assert(false)
+		return false
+
+	func _get(property: StringName) -> Variant:
+		if property == &"surname":
+			# Should not be reached.
+			assert(false)
+		elif property == &"password":
+			return 2137
+		return null
+
+	func _validate_property(property: Dictionary) -> void:
+		if property["name"] == "exported_property":
+			property["usage"] |= PROPERTY_USAGE_ARRAY
+		elif property["name"] == "surname":
+			# Dynamic property, should not be validated.
+			assert(false)
+
+	func _property_can_revert(property: StringName) -> bool:
+		if property == &"fake":
+			assert(false)
+		return false
+
+	func _property_get_revert(property: StringName) -> Variant:
+		if property  == &"fake":
+			assert(false)
+		return null
+
+class B extends A:
+	func _notification(what: int) -> void:
+		if what == 15:
+			print("B")
+
+	func _get_property_list() -> Array[Dictionary]:
+		var ret: Array[Dictionary]
+		return ret
+
+	func _set(property: StringName, value: Variant) -> bool:
+		if property == &"password":
+			prints("Setting password to:", value)
+			return true
+		return false
+
+	func _get(property: StringName) -> Variant:
+		if property == &"surname":
+			return "B-man"
+		return null
+
+	func _validate_property(_property: Dictionary) -> void:
+		pass
+
+	func _property_can_revert(property: StringName) -> bool:
+		return property == &"fake"
+
+	func _property_get_revert(property: StringName) -> Variant:
+		if property == &"fake":
+			return "fake"
+		return null
+
+class C extends B:
+	@export var exported_property: int
+
+	func _notification(what: int) -> void:
+		if what == 10:
+			print("C")
+
+	func _get_property_list() -> Array[Dictionary]:
+		var ret: Array[Dictionary]
+		ret.append({ "name": "surname", "type": TYPE_STRING })
+		return ret
+
+	func _set(_property: StringName, _value: Variant) -> bool:
+		return false
+
+	func _get(_property: StringName) -> Variant:
+		# Returning null allows to call super method.
+		return null
+
+	func _validate_property(property: Dictionary) -> void:
+		if property["name"] == "exported_property":
+			property["type"] = TYPE_ARRAY
+
+
+	func _property_can_revert(_property: StringName) -> bool:
+		return false
+
+	func _property_get_revert(_property: StringName) -> Variant:
+		print("Get revert C")
+		return null
+
+func test():
+	var c := C.new()
+
+	c.notification(10)
+	c.notification(15)
+
+	# Uses var_to_str() for better formatting.
+	print(var_to_str(c.get_property_list()))
```

Comment:
Please use `Utils.get_property_signature()` or `Utils.print_property_extended_info()` instead:

https://github.com/godotengine/godot/blob/7864ac80192e9e91bf56176af9f04cc013b580aa/modules/gdscript/tests/scripts/utils.notest.gd#L55

https://github.com/godotengine/godot/blob/7864ac80192e9e91bf56176af9f04cc013b580aa/modules/gdscript/tests/scripts/utils.notest.gd#L113

This produces a more readable and compact output. Like here:

https://github.com/godotengine/godot/blob/7864ac80192e9e91bf56176af9f04cc013b580aa/modules/gdscript/tests/scripts/runtime/features/member_info.gd#L75-L77

Reply:
I used `print_property_extended_info()`, but it spams
```
Missing `PROPERTY_USAGE_SCRIPT_VARIABLE` flag.
```
The results is usable though.